src/worker/scheduler/Task.js

```diff
@@ -8,10 +8,10 @@ class Task {
 	/**
 	 * Initializes a task.
 	 *
-	 * @param {number|string} id      The task identifier.
-	 * @param {function}      execute Executes the job with the data.
-	 * @param {function}      done    Callback for the scheduler.
-	 * @param {Object}        [data]  Optional data for when executing the task.
+	 * @param {number}   id      The task identifier.
```

Comment:
So it's been decided that `id` can't be a number anymore? If so, the check and the error message should also be changed.